src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -125,6 +129,10 @@ void* AppleseedRenderer::GetInterface(ULONG id)
 
         return static_cast<IInteractiveRender*>(m_interactive_renderer);
     }
+    if (id == IRenderElementCompatible::IID)
+    {
+        return new AppleseedRECompatible();
```

Comment:
Unline `AppleseedInteractiveRender` above, here we are returning a new instance each time. Is this intended?

Reply:
This class is only used to check element compatibility. But new instance each time is not needed, that's true. Also I don't know if it's get deleted anywhere later.